TranchedLockupPresale/migrations/2_deploy_contracts.js

```diff
@@ -0,0 +1,57 @@
+const TranchedLockupPresale = artifacts.require('./TranchedLockupPresale.sol');
+const CustomToken = artifacts.require('./CustomToken.sol');
+
+let currentTime = Math.floor(+new Date()/1000); // UNIX current timestamp in seconds
+module.exports = function(deployer, network, accounts) {
+    const openingTime = currentTime; // epoch timestamp of deployment
+    const closingTime = currentTime + 604800 // + 1 week
+
+    // //be careful! JavaScript months are 0 based: Jan = 0, Jun = 5
+    const startingYear = 2018
+    const lockupEndsTime = [
+      (new Date(startingYear,11,31,23,59,59,0)).getTime()/1000, //31 Dec 2018 23:59:59 CET
+      (new Date(startingYear + 1,5,30,23,59,59,0)).getTime()/1000, //30 Jun 2019 23:59:59 CET
+      (new Date(startingYear + 1,11,31,23,59,59,0)).getTime()/1000, //31 Dec 2019 23:59:59 CET
+      (new Date(startingYear + 2,5,30,23,59,59,0)).getTime()/1000, //30 Jun 2020 23:59:59 CET
+      (new Date(startingYear + 2,11,31,23,59,59,0)).getTime()/1000, //31 Dec 2020 23:59:59 CET
+      (new Date(startingYear + 3,5,30,23,59,59,0)).getTime()/1000 //30 Jun 2021 23:59:59 CET
+    ];
+
+    //for testing
+    // const openingTime = web3.eth.getBlock('latest').timestamp + 60;  // use it for testing
+    // const closingTime = openingTime + 60*20; // use it for testing
+    // const lockupEndsTime = [closingTime + 60*10, closingTime + 60*20, closingTime + 60*30, closingTime + 60*40, closingTime + 60*50, closingTime + 60*60];
+
+    const firstVestedLockUpAmount = new web3.BigNumber(875000e18);   //875k tokens
+    const gradedVestedLockUpAmounts = new web3.BigNumber(1000000e18);  // 1m tokens
+
+    const rate = new web3.BigNumber(1012); //per 1 ether
+    const wallet = "0x6Aee4977C8D8F8947f0784Fe934D48C677FA14A1" //  CustomToken wallet;

```

Comment:
let's change this to another wallet address (this is CareerChain's address)

Reply:
29: cont wallet = address(0) // change this to your desired wallet to receive the CustomTokens
?

Reply:
Perfect, if you forget to change it, it won't deploy. Like that.